modules/layers/src/bitmap-layer/bitmap-layer-fragment.js

```diff
@@ -66,6 +94,8 @@ vec2 getUV(vec2 pos) {
   );
 }
 
+${packUVsIntoRGB}
```

Comment:
just put the code in here

Reply:
I wanted to move this out to a pack module. General packing is very useful and tricky to get right,